apps/src/templates/manageStudents/ManageStudentsLoginInfo.jsx

```diff
@@ -166,20 +221,10 @@ class ManageStudentsLoginInfo extends Component {
         {loginType === SectionLoginType.lti_v1 && (
           <LtiLogins sectionProviderName={this.props.sectionProviderName} />
         )}
-        <h2 style={styles.heading}>{i18n.privacyHeading()}</h2>
-        <p id="uitest-privacy-text">{i18n.privacyDocExplanation()}</p>
-        <DownloadParentLetter
-          sectionId={this.props.sectionId}
-          buttonMetricsCategory={ParentLetterButtonMetricsCategory.BELOW_TABLE}
+        <GlobalRegionWrapper
+          component={ParentLetterAndStudentPrivacyInfo}
+          componentId="ParentLetterAndStudentPrivacyInfo"
```

Comment:
We remove parental letters and parental permission notices for the farsi region.